CanDIG/candigv2-ingest
# Sonchau/moh ingest

What's new:
- Update README with instructions to run the new ingest
- Add new moh_ingest, eventually will replace katsu_ingest, katsu_stats, katsu_clean
- The new ingest can:
1. Sanity check. Inspired by katsu_stats, it helps someone unfamiliar with the tool go through the checklist. More importantly, the version check makes sure all the repositories can work together. 
2. Ingest data: can pull data from local or remote. file_mapping is hardcoded but it doesn't require all the files to exist in case of some outputs are missing. The ingest still fail if missing critical components.
3. Clean data

Todo:
- Some variables are hardcode for now, e.g. repo version, remote URL

How to test:
- Make sure katsu is up to date, currently https://github.com/CanDIG/katsu/pull/100
- To use synthetic data, comment out `data_location = os.environ.get("MOH_DATA_LOCATION")` and use the `data_location` that points to github instead
- Run `python moh_ingest.py`
